packages/boot/src/booters/model.booter.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {config, Constructor, inject} from '@loopback/context';
+import {Application, CoreBindings} from '@loopback/core';
+import {ModelMetadataHelper} from '@loopback/repository';
+import debugFactory from 'debug';
+import {BootBindings} from '../keys';
+import {ArtifactOptions, booter} from '../types';
+import {BaseArtifactBooter} from './base-artifact.booter';
+
+const debug = debugFactory('loopback:boot:model-booter');
+
+/**
+ * A class that extends BaseArtifactBooter to boot the 'Model' artifact type.
+ *
+ * Supported phases: configure, discover, load
+ *
+ * @param app - Application instance
+ * @param projectRoot - Root of User Project relative to which all paths are resolved
+ * @param bootConfig - Model Artifact Options Object
+ */
+@booter('models')
+export class ModelBooter extends BaseArtifactBooter {
+  constructor(
+    @inject(CoreBindings.APPLICATION_INSTANCE)
+    public app: Application,
+    @inject(BootBindings.PROJECT_ROOT) projectRoot: string,
+    @config()
+    public modelConfig: ArtifactOptions = {},
+  ) {
+    super(
+      projectRoot,
+      // Set Model Booter Options if passed in via bootConfig
+      Object.assign({}, ModelDefaults, modelConfig),
+    );
+  }
+
+  /**
+   * Uses super method to get a list of Artifact classes. Boot each file by
+   * creating a DataSourceConstructor and binding it to the application class.
+   */
+  async load() {
+    await super.load();
+
+    for (const cls of this.classes) {
+      if (!isModelClass(cls)) {
+        debug('Skipping class %s - no @model is found', cls.name);
+        continue;
+      }
+
+      debug('Bind class: %s', cls.name);
+      // We are binding the model class itself
+      const binding = this.app.bind(`models.${cls.name}`).to(cls).tag('model');
```

Comment:
@raymondfeng can you please extract the code building the binding to a standalone function that can be used to bind models manually, in cases where `@loopback/boot` is not involved? For example in extensions.

```ts
const binding = createModelClassBinding(cls);
app.add(binding);
```

Similar helpers we already have: `createComponentApplicationBooterBinding`, `createBindingFromClass`, `createBodyParserBinding`, `createServiceBinding`.